app/serializers/category_serializer.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class CategorySerializer
+  include JSONAPI::Serializer
+
+  set_type :categories
+
+  attributes :key, :title, :move_supported
```

Comment:
After stealing this file for my PR, I made a slight tweak to use the standard `id` column, moving `key` to an attribute - I think that's a hangover from the original implementation of this endpoint that was creating a category on the fly essentially without a persisted UUID. Also added timestamp attributes and rewrote spec to match. See: https://github.com/ministryofjustice/hmpps-book-secure-move-api/pull/1111/files#diff-f3ca654ba2e5de8b9e9882a75527c0ead45a85466759a7b7a242b43c408bd1c5R8

Reply:
Actually looks like you've made the same change now, so they're pretty much in sync - just the timestamps missing from this version.

Reply:
OK, thanks -  I think I'll merge this now and let your PR add created_at, updated_at

Reply:
sounds good 👍